packages/dds/sequence/src/sequence.ts

```diff
@@ -487,10 +488,9 @@ export abstract class SharedSegmentSequence<T extends MergeTree.ISegment>
      * {@inheritDoc @fluidframework/shared-object-base#SharedObject.loadCore}
      */
     protected async loadCore(storage: IChannelStorageService) {
-        const header = await storage.read(snapshotFileName);
-
-        const data: string = header ? fromBase64ToUtf8(header) : undefined;
-        this.intervalMapKernel.populate(data);
+        const blob = await storage.readBlob(snapshotFileName);
+        const header = blobToString(blob);
+        this.intervalMapKernel.populate(header);
```

Comment:
same question here as in other places - this.intervalMapKernel.populate likel calls json.parse or serializer.parse - can we move this code up here and replace 3 steps with simple call to deserializeBlob?